website/views/static/community-guidelines.jade

```diff
@@ -30,14 +30,14 @@ block content
         |&nbsp;
         =env.t('February')
         |&nbsp;9&comma; 2015
-      h2=env.t('commGuideHeadingWelcome')
+      h2(id="welcome")=env.t('commGuideHeadingWelcome')
```

Comment:
jade has a shorthand for this: `h2#welcome=env.t('commGuideHeadingWelcome')`

Would you mind making that change every place you've added ids?